moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1019,11 +1019,9 @@ const Eigen::Isometry3d& RobotState::getFrameInfo(const std::string& frame_id, c
     frame_found = true;
     return IDENTITY_TRANSFORM;
   }
-  if (robot_model_->hasLinkModel(frame_id))
+  if ((robot_link = robot_model_->getLinkModel(frame_id, &frame_found)))
```

Comment:
nit: Remove the outermost bracket?

Reply:
The extra pair of brackets is required to interpret the assignment result as a boolean (and silent compiler warnings, because this might be wrongly typed `==` comparison as well.)

Reply:
Interesting. Thanks.